lib/src/cli.rs

```diff
@@ -749,6 +763,18 @@ async fn container_export(
         let buf = File::open(contentmeta).map(BufReader::new);
         let raw: RawMeta = serde_json::from_reader(buf?)?;
 
+        // Check future variables are set correctly
+        if let Some(version) = raw.version {
+            if version != 1 {
+                return Err(anyhow::anyhow!("Unsupported metadata version: {}", version));
+            }
+        }
```

Comment:
If we want people to put a version in there, shouldn't we just hard require it?

Reply:
I did not want to require it because there might never be a v2, so v1 may as well not have it.

However, if there is a V2, this check will fail if the wrong ostree-ext-cli is used.

Although come to think about it, the compiler will whine either way because of other config changes.

I can make it mandatory if you want though.

Reply:
Mild preference for requiring it but not strongly held

Reply:
Made mandatory.
```
Creating archive with ref oci:oci_rerechunk15.2
error: missing field `version` at line 190133 column 1

Creating archive with ref oci:oci_rerechunk15.2
error: Unsupported metadata version: 2. Currently supported: 1
```